src/components/ListItemWrapper.tsx

```diff
@@ -66,7 +74,7 @@ const ListItemWrapper = (props: ListItemWrapperProps) => {
   const getRightText = () =>
   (<ListItemText
     className={classes.rightTextStyles}
-    primaryTypographyProps={{ align: 'right' }}
+    primaryTypographyProps={{ align: 'right', variant: 'h4' }}
```

Comment:
This might break other uses of this component. We should pass this as an optional prop like `rightTextVariant` or something 

This makes the settings page right text bolder than on Figma
![image](https://user-images.githubusercontent.com/35501399/115969334-42386d80-a4f1-11eb-9227-7ab42f9a2151.png)